playwright-tests/tests/funding.spec.js

```diff
@@ -11,24 +15,35 @@ test.describe("Wallet is connected", () => {
     //
     page,
   }) => {
+    test.setTimeout(60000);
     await page.goto("/devhub.near/widget/app?page=create");
 
     await page.click('button:has-text("Solution")');
+    await pauseIfVideoRecording(page);
 
-    await page.getByTestId("name-editor").fill("The test title");
+    const titlefield = await page.getByTestId("name-editor");
+    expect(titlefield).toBeVisible();
+    await titlefield.scrollIntoViewIfNeeded();
+    await titlefield.fill("The test title");
+    await pauseIfVideoRecording(page);
 
     const descriptionInput = page
       .frameLocator("iframe")
       .locator(".CodeMirror textarea");
     await descriptionInput.focus();
+    await descriptionInput.scrollIntoViewIfNeeded();
     await descriptionInput.fill("Developer contributor report by somebody");
+    await descriptionInput.blur();
+    await pauseIfVideoRecording(page);
 
     const tagsInput = page.locator(".rbt-input-multi");
+    await tagsInput.scrollIntoViewIfNeeded();
     await tagsInput.focus();
     await tagsInput.pressSequentially("paid-cont", { delay: 100 });
     await tagsInput.press("Tab");
     await tagsInput.pressSequentially("developer-da", { delay: 100 });
     await tagsInput.press("Tab");
+    await pauseIfVideoRecording(page);
```

Comment:
I don't understand, why do we check it after every step? shouldn't we check only once if video exists.

Reply:
The `pauseIfVideoRecording` method checks if you have set `video` to `on` in your playwright.config.js here:

<img width="384" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-bos/assets/9760441/08240ffa-5815-4cbd-a471-2ada0d516ace">

If video recording is enabled, then the `pauseIfVideoRecording` method will just insert a 0.5 second delay before moving on with the next step. This is just so that the video should be a bit easier to watch, and not running too fast. In the pipeline we want as few delays as possible, and there video recording should be set to `off`, and there will be no delay.